Dockerfile

```
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+COPY . /app/
+WORKDIR /app
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+RUN $HOME/.poetry/bin/poetry config virtualenvs.in-project true
+RUN $HOME/.poetry/bin/poetry install
+RUN .venv/bin/python manage.py migrate
+RUN $HOME/.poetry/bin/poetry build
+CMD [".venv/bin/python", "manage.py", "runserver", "0.0.0.0:80"]
```

프로덕션 환경에선 runserver를 권장하지 않는걸로 알고 있습니다. [참고](https://stackoverflow.com/questions/23176592/deploying-django-by-python-manage-py-runserver-to-production-on-vps)

`scripts/on-isteam.sh` 참고해서 다음처럼 수정하는게 좋을것같아요.
```suggestion
CMD gunicorn isteam.wsgi:application --bind 0.0.0.0:8000
```

---

scripts/on-isteam을 조금씩 조작하면서 gunicorn 사용법을 익힐려고 했는데 mac 환경에서는 실행 오류가 나네요.
혹시 gunicorn 사용법 대략적으로 알 수 있을까요?